jazkarta/shop/utils.py

```diff
@@ -30,8 +38,8 @@ def get_site():
         possible_site = aq_parent(possible_site)
     return possible_site
 
-
-get_navigation_root_url = get_portal_url
+def get_navigation_root_url():
+    return getSite().absolute_url()
```

Comment:
We should probably use `plone.api.portal.get_navigation_root().absolute_url()` so that it works right in sites using subsites with collective.lineage

Now that we are requiring plone.api there are probably some other functions in utils.py that should use plone.api rather than reinventing the wheel.